packages/react-native-editor/package.json

```diff
@@ -114,8 +114,8 @@
     "test:e2e:bundle:android": "mkdir -p android/app/src/main/assets && react-native bundle --reset-cache --platform android --dev false --minify false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res",
     "test:e2e:build-app:android": "npm run test:e2e:bundle:android && cd android && ./gradlew clean && ./gradlew assembleDebug",
     "test:e2e:install-app:android": "cd android && ./gradlew installDebug",
-    "test:e2e:bundle:ios": "react-native bundle --reset-cache --platform=ios --dev=false --minify false --entry-file=index.js --bundle-output=./ios/build/gutenberg/Build/Products/Release-iphonesimulator/GutenbergDemo.app/main.jsbundle --assets-dest=./ios/build/gutenberg/Build/Products/Release-iphonesimulator/GutenbergDemo.app",
-    "test:e2e:build-app:ios": "npm run ios --configuration Release --no-packager",
+    "test:e2e:bundle:ios": "mkdir -p ios/build/gutenberg/Build/Products/Release-iphonesimulator/GutenbergDemo.app && react-native bundle --reset-cache --platform=ios --dev=false --minify false --entry-file=index.js --bundle-output=./ios/build/gutenberg/Build/Products/Release-iphonesimulator/GutenbergDemo.app/main.jsbundle --assets-dest=./ios/build/gutenberg/Build/Products/Release-iphonesimulator/GutenbergDemo.app",
```

Comment:
Could we call this script from `test:e2e:ios:local`? Otherwise we have to call it manually each time

Reply:
Bundling should automatically happen when `test:e2e:build-app:ios` is called via a script in the iOS Xcode project. In the CI case we call them seperately for caching purposes and add `SKIP_BUNDLING=true` flag before calling `test:e2e:build-app:ios`

Reply:
Agreed for automated CI using github actions. But for manually running e2e tests locally (which is what `test:e2e:ios:local` is used for atm) we need to automatically build the js bundle. I'll add it

Reply:
I wanted to say it was already happening during `test:e2e:build-app:ios` locally. There's this script in Xcode: 
https://github.com/facebook/react-native/blob/0.61-stable/scripts/react-native-xcode.sh

<img width="697" alt="xcode" src="https://user-images.githubusercontent.com/1845482/82034117-0caec180-969e-11ea-8b93-97d0b5caab82.png">


This is invoked by Xcode via `react-native run-ios` when `test:e2e:build-app:ios` is called and should create the bundle already. 
But now it returns an error saying `gutenberg/index.js was not found`. Maybe that's why `test:e2e:ios:local` was returning an error locally .

If we create the bundle before `react-native-xcode.sh` is invoked, it exits [without throwing an error](https://github.com/facebook/react-native/blob/0.61-stable/scripts/react-native-xcode.sh#L127) 
So it's fine, for now 🙂